src/components/ProjectForm/ProjectDetail.jsx

```diff
@@ -0,0 +1,76 @@
+import axios from "axios";
+import { useParams } from "react-router-dom";
+import {
+  Button,
+  Container,
+  Divider,
+  Grid,
+  Typography,
+} from "@material-ui/core";
+import React, { useEffect, useState } from "react";
+const Project = () => {
```

Comment:
Please leave an empty line between imports and the rest of the code

Reply:
This part is Solved